dimod/quadratic/quadratic_model.py

```diff
@@ -899,17 +899,17 @@ def lower_bound(self, v: Variable) -> Bias:
         """
         return self.data.lower_bound
 
-    def set_lower_bound(self, v: Variable, lb: int):
-        """Set the lower bound for an integer variable.
+    def set_lower_bound(self, v: Variable, lb: float):
+        """Set the lower bound for a variable.
```

Comment:
Maybe we should update this to say: "Sets the lower bound for a variable with modifiable bounds" here and below; since it should throw an exception in the case of binary/spin.

Reply:
I'll add a `Raises:` section.